tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -623,3 +625,64 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+def validate__WORK_ELIGIBLE_INDICATOR__HOH__AGE():
+    """If WORK_ELIGIBLE_INDICATOR == 11 and AGE < 19, then RELATIONSHIP_HOH != 1."""
+    # value is instance
+    def validate(instance):
+        false_case = (False,
+                      "If WORK_ELIGIBLE_INDICATOR == 11 and AGE < 19, then RELATIONSHIP_HOH != 1",
+                      ['WORK_ELIGIBLE_INDICATOR', 'RELATIONSHIP_HOH', 'DATE_OF_BIRTH']
+                      )
+        true_case = (True,
+                     None,
+                     ['WORK_ELIGIBLE_INDICATOR', 'RELATIONSHIP_HOH', 'DATE_OF_BIRTH'],
+                     )
+        try:
+            WORK_ELIGIBLE_INDICATOR = (
+                instance["WORK_ELIGIBLE_INDICATOR"]
+                if type(instance) is dict
+                else getattr(instance, "WORK_ELIGIBLE_INDICATOR")
+            )
+            RELATIONSHIP_HOH = (
+                instance["RELATIONSHIP_HOH"]
+                if type(instance) is dict
+                else getattr(instance, "RELATIONSHIP_HOH")
+            )
+            RELATIONSHIP_HOH = int(RELATIONSHIP_HOH)
+
+            DOB = str(
+                instance["DATE_OF_BIRTH"]
+                if type(instance) is dict
+                else getattr(instance, "DATE_OF_BIRTH")
+            )
+
+            RPT_MONTH_YEAR = str(
+                instance["RPT_MONTH_YEAR"]
+                if type(instance) is dict
+                else getattr(instance, "RPT_MONTH_YEAR")
+            )
+
+            RPT_MONTH_YEAR += "01"
+
+            DOB_datetime = datetime.datetime.strptime(DOB, '%Y%m%d')
+            RPT_MONTH_YEAR_datetime = datetime.datetime.strptime(DOB, '%Y%m%d')
+            AGE = (RPT_MONTH_YEAR_datetime - DOB_datetime).days / 365.25
```

Comment:
@elipe17 i'm not seeing the error message for this one. 

![2681p1](https://github.com/raft-tech/TANF-app/assets/63075587/58191cb0-1f44-408e-9e72-00e15e604dfd)

Reply:
@ADPennington, there was a type error in the validator. I have resolved and the error will generate now